compiler/p/codegen/OMRCodeGenerator.cpp

```diff
@@ -1873,6 +1873,10 @@ bool OMR::Power::CodeGenerator::getSupportsOpCodeForAutoSIMD(TR::CPU *cpu, TR::I
       case TR::vcmpge:
          if (et == TR::Double)
             return true;
+         else
+            return false;
```

Comment:
A bit unrelated but can you add this fix to `TR::vconv` as well? `TR::vconv` also needs a false path.